components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -153,6 +171,14 @@ public void handleEvent(Event event) throws IdentityEventException {
                 }
             }
         }
+
+        if (IdentityEventConstants.Event.PRE_SET_USER_CLAIMS.equals(eventName)) {
+            preSetUserClaimsOnEmailUpdate(claims, userStoreManager, user);
+        }
+
```

Comment:
Shall we remove this line?

Reply:
Added the line here to improve the readability.